### PR TITLE
Added Spanish translation

### PR DESCRIPTION
**Pull request by javierfdezg**

### CODE REVIEW

**Comment by ecunado** [comment]
Good translation!

**Comment by SamirTalwar** [comment]
Thanks so much!

It's up at https://es.communitycodeofconduct.com/.

**Comment by mavelezserna** [comment]
It's good and clear. Couple of suggestions:

1. I assume you've chosen not to use a/@/x  endings to make nouns gender-inclusive? For instance "todos y todas", los/las organizadores/as, etc. It's clunky and I don't know what the current best practice is. But thought it was worth raising.

2. I'd suggest "No toleramos ningún tipo de acoso..." Rather than "No toleramos el acoso (...) cualquiera que sea su forma" 

3. "contacta con un(a) organizador(a)" should be either "ponte en contacto con..," o "contacta a..."

File: index-es.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+<p>Acoso incluye comentarios verbales ofensivos relacionados con g&eacute;nero, identidad y expresi&oacute;n de g&eacute;nero, edad, orientaci&oacute;n sexual, discapacidad, apariencia f&iacute;sica, tama&ntilde;o corporal, raza, etnia, religion, preferencias tecnol&oacute;gicas, im&aacute;genes de &iacute;ndole sexual en espacios p&uacute;blicos, intimidaci&oacute;n intencionada, acechar, seguir, toma de fotograf&iacute;as o grabaci&oacute;n de video de una manera hostigadora, interrupci&oacute;n continuada de ponencias u otros eventos, contacto f&iacute;sico inapropiado y proposiciones sexuales no deseadas.</p>
+
+<p>Se espera que aquellos participantes a los que se les solicite cesar cualquier tipo de acoso lo hagan de manera inmediata.</p>
+
+<p>Los esp&oacute;nsors tambi&eacute;n est&aacute; sujetos a la pol&iacute;tica anti acoso. En particular, los esp&oacute;nsors no deber&aacute; utilizar im&aacute;genes, movimientos u otro material sexualizado. Los anfitriones y asistentes, incluyendo a los esp&oacute;nsors, empleados y voluntarios, no deber&aacute;n utilizar ropa/uniformes/disfraces sexualizados o crear un ambiente sexualizado de cualquier otra manera.</p>
```
Comment (islomar) [inline index-es.html:84]: "no deberán" instead of "no deberá"

**Comment by javierfdezg** [comment]
Hey @mavelezserna! 

1) I Spanish, the form I used is grammatically correct: use the masculin as gender-inclusive noun. [Here](http://www.rae.es/consultas/los-ciudadanos-y-las-ciudadanas-los-ninos-y-las-ninas) and [here](http://www.fundeu.es/noticia/lenguaje-inclusivo-6151/) are some references. Using the terminations you mention is not needed if we follow the norm.

2) Good point! I'll update it.

3) Will also update it.

Thanks for your suggestions!

**Comment by SamirTalwar** [comment]
I don't speak any Spanish at all, so feel free to ignore me, but I thought it was worth pointing out that English is making a slow transition back to using "they"/"them" for gender-inclusivity, especially when it comes to non-binary people. From what I've read, the "x" suffix represents the same thing in Spanish (especially in Latin America).

I don't know what the right solution is either, and I think it's probably more sensible to lean towards a translation that people can read easily. English has the advantage here because "they" and "them" have informally been used as gender-unknown pronouns for centuries, so dropping them in causes no problems.

In short: whatever you folks decide between yourselves is fine by me.

File: index-es.html (hunk 1 above)
Comment (javierfdezg) [reply index-es.html:84]: Thanks! This was a typo :) Fixed in the next PR

**Comment by javierfdezg** [comment]
The 'x' is currently being used to avoid using masculin or feminine (eg: niños/niñas = niñxs). I focused on doing this document as readable as I could. IMO is good as is. Please feel free to change/modify any excerpt :)

File: index-es.html (hunk 1 above)
Comment (jonasanso) [inline index-es.html:84]: Good transalation. 
Only one question. "imagenes, movimientos u otro material sexualizado"  Movimientos sounds a bit strange here, why not use "actividades" ?

Comment (javierfdezg) [reply index-es.html:84]: You are right... thanks!

**Comment by javierfdezg** [comment]
Closed this PR, please have a look at the fixes and further suggestions [here](https://github.com/SamirTalwar/communitycodeofconduct.com/pull/16).
